src/app/containers/Timestamp/index.test.jsx

```diff
@@ -1,25 +1,68 @@
 import React from 'react';
+import { render } from 'enzyme';
 import { isNull, shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import timestampGenerator from './helpers/timestampGenerator';
 import Timestamp from '.';
 
 const timestamp = 1539969006000; // 19 October 2018
 const noLeadingZeroTimestamp = 1530947227000; // 07 July 2018
 const invalidTimestamp = 8640000000000001; // A day holds 86,400,000 milliseconds - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Description
 
+const eighthMarch = 1552009884472; // 2019-03-08T01:51:24.472Z
+const fifthJan = 1546707084472; // 2019-01-05T16:51:24.472Z
+
 describe('Timestamp', () => {
   describe('with no data', () => {
     isNull('should return null', <Timestamp />);
   });
   shouldMatchSnapshot(
     'should render without a leading zero on the day',
-    <Timestamp timestamp={noLeadingZeroTimestamp} />,
+    <Timestamp
+      updated={noLeadingZeroTimestamp}
+      published={noLeadingZeroTimestamp}
+    />,
   );
   shouldMatchSnapshot(
     'should render correctly',
-    <Timestamp timestamp={timestamp} />,
+    <Timestamp updated={timestamp} published={timestamp} />,
   );
   shouldMatchSnapshot(
     'should handle an invalid timestamp',
-    <Timestamp timestamp={invalidTimestamp} />,
+    <Timestamp updated={invalidTimestamp} published={invalidTimestamp} />,
   );
+
+  it('should display only one timestamp when published === updated', () => {
+    const renderedWrapper = render(
+      <Timestamp updated={fifthJan} published={fifthJan} />,
+    );
+    expect(renderedWrapper.get(0).children[0].data).toEqual('5 January 2019');
+    expect(renderedWrapper.length).toEqual(1);
+  });
+
+  it('should display a relative timestamp when updated < 10 hours ago', () => {
+    const sixHoursAgo = timestampGenerator({ hours: 6 });
+    const renderedWrapper = render(
+      <Timestamp updated={sixHoursAgo} published={fifthJan} />,
+    );
+
+    expect(renderedWrapper.length).toEqual(2);
```

Comment:
I agree with chris about it being a bit low level, the toString solution seems nice in patrnership with the snapshot tests you have

Reply:
`expect(renderedWrapper.text()).toEqual('5 January 2019Updated 6 hours ago');`
^ this is as close as it's going to get to using `toString`
You can take a look at the [Enzyme API methods available](https://airbnb.io/enzyme/docs/api/mount.html) but if you check my comment above I have tried most if not all of them and they don't work because this wrapper contains two nodes.

Reply:
Ah strange, Ile pull the branch and take a look when im in the office :)

Reply:
I'm fairly happy with these but if you can find a way to make them better, that'd be great! 👍